packages/integration/config.d.ts

```diff
@@ -15,7 +15,10 @@
  */
 
 export interface Config {
-  /** Configuration for integrations towards various external repository provider systems */
+  /**
+   * Configuration for integrations towards various external repository provider systems
+   * @visibility frontend
```

Comment:
Hmm, this isn't gonna do much really, it won't make for example `integrations.azure` visible. We'd need separate visibilities on each of the different integrations

Reply:
yes, it would only make sure that `integrations` itself is visible which is enough for the catalog-import plugin.

In order to make e.g. `integrations.bitbucketCloud` visible which has no frontend visible properties like with
```
integrations:
  bitbucketCloud:
    - username: any-user
       appPassword: any-password
```

We would need to apply the same to `integrations.bitbucketCloud` (and maybe all others, too). Still, the frontend would not see anything else as nothing below is visible to it.

In general, this depends on the integration and its properties and their visibility.

`integrations.azure` would have worked here as `integrations.azure[].host` is visible to the frontend.

Currently, this could have happened for the integrations:

- bitbucketCloud
- googleGcs